float64[500] A1
float64[500] A2
float64[500] A3
float64[500] A4
float64[500] A5
float64[500] A6
float64[500] A7
float64[500] A8

